README.md

```diff
@@ -542,6 +542,16 @@ amadeus.dutyOfCare.diseases.covid19Report.get({
   language: 'EN'
 });
 
+//Car & Transfers APIs
+// Transfer Search API : Search Transfer offers 
```

Comment:
I would suggest to write it as "Transfer Search API: Search Transfer". usually there is only space at the right side. Same for the Management and Booking